builtin/mainmenu/dlg_version_info.lua

```diff
@@ -0,0 +1,125 @@
+--Minetest
+--Copyright (C) 2018 SmallJoker
+--
+--This program is free software; you can redistribute it and/or modify
+--it under the terms of the GNU Lesser General Public License as published by
+--the Free Software Foundation; either version 2.1 of the License, or
+--(at your option) any later version.
+--
+--This program is distributed in the hope that it will be useful,
+--but WITHOUT ANY WARRANTY; without even the implied warranty of
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU Lesser General Public License for more details.
+--
+--You should have received a copy of the GNU Lesser General Public License along
+--with this program; if not, write to the Free Software Foundation, Inc.,
+--51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+
+local function version_info_formspec(data)
+	return (
+		"size[9,4.5,true]" ..
+		"textarea[0.5,0.5;8.5,3;;" ..
+		fgettext("A new Minetest version was found!") .. ";" ..
+		fgettext("Current version: $1\nNew version: $2\n" ..
+			"Visit $3 to find out how to get the newest version to stay up to date" ..
```

Comment:
Adding another new line before `Visit $3 to...` will make it more readable, I think